specification/maps/data-plane/Common/preview/2022-09-01-preview/common.json

```diff
@@ -158,9 +158,8 @@
     },
     "BoundingBox": {
       "x-ms-client-name": "boundingBox",
-      "description": "Bounding box. Projection used - EPSG:3857. Format : 'minLon, minLat, maxLon, maxLat'.",
+      "description": "Bounding box. Projection used - EPSG:3857. Please refer to [RFC 7946](https://datatracker.ietf.org/doc/html/rfc7946#section-5) for details.",
       "type": "array",
-      "readOnly": true,
```

Comment:
I see removing of the `readOnly` here. I assume it does not have effect, as `readOnly` should be in property, not in type.